### ISSUE TITLE
please fix paymentInstrument type by making it nullable

### ISSUE THREAD

**Issue by naseemkullah** [Adyen/adyen-node-api-library#1323]
https://github.com/Adyen/adyen-node-api-library/blob/7d0a34e6df7e56c6b6d3124a2982deb2182f21e2/src/typings/management/payAtTable.ts#L23

To have both cash and card as payment instruments, we must be able to unset any enum value (card or cash) that had been set by sending `null`. Please change the type to:

```
'paymentInstrument'?: PayAtTable.PaymentInstrumentEnum | null;
```

**Comment by jillingk** [comment]
Hey @naseemkullah,

Thanks for bringing this to our attention. I just opened a quick PR to fix your issue and expect the next release (bi-weekly) to include said fix.

Best, Jilling
Adyen

**Comment by naseemkullah** [comment]
amazing, thanks for the quick turnaround @jillingk

**Comment by naseemkullah** [comment]
Hi @jillingk, seems like type is not updated in 16.0.2, still getting ` 'paymentInstrument'?: PayAtTable.PaymentInstrumentEnum;` without the `| null`

**Comment by naseemkullah** [comment]
fixed in 16.1.0
